docs/Storage/Storage_best_practices.md

```diff
@@ -0,0 +1,60 @@
+# Storage Best Practises
+
+This page will guide you through best practices for storing your data on Mahuika
+
+## The 3,2,1 Rule
+
+The 3,2,1 rule is a best practise of backing up data to minimise lose of mission critical data. The rule is
+
+* **3 Copies**: Keep your original data plus two backups.
+* **2 Media Types**: Store backups on different media (e.g., Internal Hard Drive, External Hard Drives, Cloud, Network Attached Device, High Capacity Storage).
+* **1 Offsite**: Keep one copy physically separate (e.g., cloud, offsite drive) to survive local disasters.
+
+## What are `home`, `project` and `nobackup` directories for?
+
+`home`: This directory is for storing files that are central for performing your simulations and calculations on Mahuika. This includes program installations, conda environments and other virtual environments.
+
+`project`: This directory is for storing files that you want to keep for long periods of time that you access regularly. This includes important research files you do not want deleted and you access regularly, and can include conda environments and other virtual environments that are too large for `home`. 
+
+`nobackup`: This directory is for storing larger files that are temporary. In this case, temporary may be for short (seconds) to long (90 days) amounts of time. Examples include `temp` files, files that you only need for less than 3 months at a time, OTHER EXAMPLES. 
```

Comment:
Maybe add a note about nobackup being a place for raw data when it is being used before being stored somewhere more permanent (like Freezer)

To me, the descriptions of project/nobackup reads like raw data should be stored in project